drexelundweiss/__init__.py

```diff
@@ -18,74 +18,90 @@
+            self._pollservice = False
 
+            self.devl[1] = {'device': 'aerosilent primus', 'cmdpath':
+                            smarthome.base_dir + '/plugins/drexelundweiss/aerosilent_primus.txt'}
+            self.devl[2] = {'device': 'aerosilent topo', 'cmdpath':
+                            smarthome.base_dir + '/plugins/drexelundweiss/aerosilent_topo.txt'}
+            self.devl[3] = {'device': 'aerosilent micro', 'cmdpath':
+                            smarthome.base_dir + '/plugins/drexelundweiss/aerosilent_micro.txt'}
+            self.devl[4] = {'device': 'aerosmart s', 'cmdpath':
+                            smarthome.base_dir + '/plugins/drexelundweiss/aerosmart_s.txt'}
+            self.devl[5] = {'device': 'aerosmart m', 'cmdpath':
+                            smarthome.base_dir + '/plugins/drexelundweiss/aerosmart_m.txt'}
+            self.devl[6] = {'device': 'aerosmart l', 'cmdpath':
+                            smarthome.base_dir + '/plugins/drexelundweiss/aerosmart_l.txt'}
+            self.devl[7] = {'device': 'aerosmart xls', 'cmdpath':
+                            smarthome.base_dir + '/plugins/drexelundweiss/aerosmart_xls.txt'}
+            self.devl[8] = {'device': 'aerosilent centro', 'cmdpath':
+                            smarthome.base_dir + '/plugins/drexelundweiss/aerosilent_centro.txt'}
+            self.devl[9] = {'device': 'termosmart sc', 'cmdpath':
+                            smarthome.base_dir + '/plugins/drexelundweiss/termosmart_sc.txt'}
+            self.devl[10] = {'device': 'x2', 'cmdpath':
+                             smarthome.base_dir + '/plugins/drexelundweiss/x2.txt'}
+            self.devl[11] = {'device': 'aerosmart mono', 'cmdpath':
+                             smarthome.base_dir + '/plugins/drexelundweiss/aerosmart_mono.txt'}
+            self.devl[13] = {'device': 'aerosilent bianco', 'cmdpath':
+                             smarthome.base_dir + '/plugins/drexelundweiss/aerosilent_bianco.txt'}
+            self.devl[14] = {'device': 'x2 plus', 'cmdpath':
+                             smarthome.base_dir + '/plugins/drexelundweiss/x2_plus.txt'}
+            self.devl[15] = {'device': 'aerosilent business', 'cmdpath':
+                             smarthome.base_dir + '/plugins/drexelundweiss/aerosilent_business.txt'}
+            self.devl[17] = {'device': 'aerosilent stratos', 'cmdpath':
+                             smarthome.base_dir + '/plugins/drexelundweiss/aerosilent_stratos.txt'}
+        except Exception as err:
+            self.logger.error("{}: Error on init {}.".format(self._name, err))
+            self._init_complete = False
+            return
         try:
-            self._port = serial.Serial(tty, 115200, timeout=5)
+            self._port = serial.Serial(self._tty, 115200, timeout=5)
         except:
-            self.logger.error("DuW: could not open {}.".format(tty))
+            self.logger.error("{}: could not open {}.".format(self._name, self._tty))
```

Comment:
Isn't the plugin logger already including the plugin name in the message?

Reply:
This part was taken from here: https://github.com/smarthomeNG/smarthome/commit/31fb104d03c6d962d3faaef9f68e8bdac96bd3e6

But it looks like this changed meanwhile.. So starting the log entry with the plugin name isn't useful anymore?

Reply:
Changed logging. Hope it is fine this way.
By the way: logutils verhindert Smarthome Start, weil in Zeile 41 das : hinter else fehlt!